lib/Popup.js

```diff
@@ -124,7 +124,10 @@ exports.default = _react2.default.createClass({
 
     if (opening) this.open();else if (closing) this.close();else if (open) {
       var height = this.height();
-      if (height !== this.state.height) this.setState({ height: height });
+      var diff = Math.abs(height - this.state.height);
+      if (isNaN(diff) || diff > 0.1) {
```

Comment:
Sure! I have added a comment and structured the surrounding if-statement a bit since this block now became pretty long. You could also handle the problem by rounding the target size returned by this.height() to the nearest integer, but I guess browsers actually have sub-pixel positioning so we might want the exact value?